### PR TITLE
STEP 3

### PR DESCRIPTION
**Pull request by am1nat**
Please check it and review

### CODE REVIEW

**Comment by am1nat** [comment]
Resolve Conversation

**Review by mendozacolonx — COMMENTED** [review]
Im not convinced about the "<h1>" code.

**Comment by am1nat** [comment]
okay i am not either

**Review by mendozacolonx — COMMENTED** [review]
test

**Comment by am1nat** [comment]
exam

File: index.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
         <!-- Navigation-->
         <nav class="navbar navbar-light bg-light static-top">
             <div class="container">
-                <a class="navbar-brand" href="#!">Start Bootstrap</a>
+                <a class="navbar-brand" href="#!">Dental Care</a>
```
Comment (mendozacolonx) [inline index.html:22]: I'm not convinced about the heading change

File: index.html [hunk 2]
```diff
@@ -19,7 +19,7 @@
         <!-- Navigation-->
         <nav class="navbar navbar-light bg-light static-top">
             <div class="container">
-                <a class="navbar-brand" href="#!">Start Bootstrap</a>
```
Comment (mendozacolonx) [inline index.html:22]: I'm not convinced about the heading change

File: index.html (hunk 1 above)
Comment (am1nat) [reply index.html:22]: okay lets check for something else

File: index.html (hunk 2 above)
Comment (mendozacolonx) [inline index.html:22]: starting a review attempt #2, please change the heading, font or color.

**Review by mendozacolonx — COMMENTED** [review]
making the review visible to team

File: index.html (hunk 2 above)
Comment (am1nat) [reply index.html:22]: we will work on that

Comment (mendozacolonx) [inline index.html:22]: attempt 3

**Comment by am1nat** [comment]
okay i will

File: index.html (hunk 2 above)
Comment (am1nat) [reply index.html:22]: alright will check it out

File: index.html (hunk 1 above)
Comment (mendozacolonx) [inline index.html:22]: testing comment

Comment (am1nat) [reply index.html:22]: comment

**Pull request merged**